src/main/java/com/autotune/analyzer/experiment/ExperimentInterfaceImpl.java

```diff
@@ -50,9 +57,27 @@ public boolean addExperimentToLocalStorage(Map<String, KruizeObject> mainKruizeE
 
     @Override
     public boolean addExperimentToDB(KruizeObject kruizeObject) {
-        //TODO insert in to db
+        boolean success = false;
         updateExperimentStatus(kruizeObject, AnalyzerConstants.ExperimentStatus.IN_PROGRESS);
-        return true;
+        ExperimentDetailTable experimentDetailTable = ServiceHelpers.Converters.KruizeObjectConverters.convertKruizeObjectToExperimentDBObj(kruizeObject);
```

Comment:
Where are we doing the error checking for this and how are the errors captured for the user to be made aware of?

Reply:
@dinogun As we mentioned in our previous call, the insertion of data into the database occurs only after the message is added to the queue. In order to provide a report to the user, we would need to implement a UUID mechanism. Currently, if there are any errors during the database insertion process, the only way to identify them is through the log. However, we can temporarily resolve this issue until the UUID system is implemented by eliminating the queue for single uploads and performing the database insertion immediately after validation. Please inform me if this solution is necessary for the 12th release.

Reply:
The queue was to be implemented only for the case when multi record arrays are fed to createExperiment/updateResults. For single records, we can insert into DB directly after completing validation checking for both createExperiment and updateResults. This is what is needed for `0.0.12_rm` release